src/lib/OS/SimOS.cc

```diff
@@ -270,5 +311,50 @@ VAddrTranslator SimOS::getVAddrTranslator() {
   return fn;
 }
 
+// The Private constructor
+SimOS::SimOS(std::shared_ptr<simeng::memory::Mem> mem)
```

Comment:
Would it be better to put all constructor near each other?

Reply:
Done